shell/src/main/java/alluxio/cli/fs/command/CountCommand.java

```diff
@@ -47,7 +59,7 @@ public String getCommandName() {
 
   @Override
   public void validateArgs(CommandLine cl) throws InvalidArgumentException {
-    CommandUtils.checkNumOfArgsEquals(this, cl, 1);
+    CommandUtils.checkNumOfArgsEquals(this, cl, 2);
```

Comment:
`-h` is optional so I think the num of args will stay 1. Can you please build Alluxio and give it a manual test?

Reply:
ok